projects/plugins/jetpack/extensions/blocks/videopress/resumable-upload/index.js

```diff
@@ -106,6 +106,39 @@ export default function ResumableUpload( { file } ) {
 
 	const fileSizeLabel = filesize( file.size );
 
+	const getErrorMessage = () => {
+		let errorMessage = __(
+			'An error was encountered during the upload. Check your network connection.',
+			'jetpack'
+		);
+		if ( typeof error === 'object' ) {
+			const apiResponse = error.toString().match( /message":"([^"]+)"/ );
```

Comment:
Is there something we could change on server side to prevent having to use this regex? We'd be happy to update it.

Reply:
I'm not sure. This is something that happens in the tus-client lib and at some point I gave up looking for it in hopes that maybe you would have the answer :D Maybe it has to do with the response code the server sends

Reply:
For future reference, the problems seems to be that the TUS protocol does not expect the response to be a json, and also the client does not handle errors on "creation" very well. See p1655822877174109-slack-CBG1CP4EN